legal-api/src/legal_api/services/authz.py

```diff
@@ -164,24 +166,21 @@ def has_roles(jwt: JwtManager, roles: List[str]) -> bool:
                 }
             },
             'courtOrder': {
-                'legalTypes': ['SP', 'GP', 'CP', 'BC', 'BEN', 'CC', 'ULC', 'LLC'],
-                'blockerChecks': {
-                    'business': [BusinessBlocker.DEFAULT]
-                }
+                'legalTypes': ['SP', 'GP', 'CP', 'BC', 'BEN', 'CC', 'ULC']
             },
             'dissolution': {
                 'voluntary': {
-                    'legalTypes': ['CP', 'BC', 'BEN', 'CC', 'ULC', 'LLC', 'SP', 'GP'],
+                    'legalTypes': ['CP', 'BC', 'BEN', 'CC', 'ULC'],
                     'blockerChecks': {
                         'warningTypes': [WarningType.MISSING_REQUIRED_BUSINESS_INFO],
-                        'business': [BusinessBlocker.DEFAULT]
+                        'business': [BusinessBlocker.DEFAULT, BusinessBlocker.GOOD_STANDING]
```

Comment:
Does this mean a user can start a Voluntary Dissolution filing as long as the business is in good standing? What does "good standing" mean -- not IA/Reg? not frozen? not historical? no other draft filings?

Reply:
This will be updated to be `BusinessBlocker.NOT_IN_GOOD_STANDING`.

Reply:
I was wondering whether all the right checks are in place for a voluntary dissolution. I read further below and it looks like DEFAULT handles the other conditions (is business, is not frozen, is in good standing).

Is "active" checked for this filing somehow?

Reply:
I believe the latest code in place should be good.  Active is already filtered for higher up in the dictionary.

<img width="799" alt="image" src="https://user-images.githubusercontent.com/6685223/231573651-2e0663f0-49de-4713-8563-384fb6448f66.png">